style-guide/google_checks.xml

```diff
@@ -41,7 +41,8 @@
 
   <module name="LineLength">
     <property name="fileExtensions" value="java"/>
-    <property name="max" value="100"/>
+    <!-- intellij standard page line -->
+    <property name="max" value="150"/>
```

Comment:
I assume, that 120 is the hard wrap as default line Length in IntelliJ (atleast in Ultimate version )

Reply:
sure ... I just think something above 100 is good for me my standard page is 150.
I feel its better to be able to not force line breaks but leave it to the developer to decide what is more readable but have a limit as not to go crazy.